packages-web/datagrid/src/components/Header.tsx

```diff
@@ -61,10 +63,10 @@ export function Header<D extends object>({
                     onClick={canSort ? onClick : undefined}
                 >
                     {column.render("Header")}
+                    {sortIcon && <FontAwesomeIcon icon={sortIcon} color="#606671" />}
```

Comment:
hmm this way developers cannot replace an icon using css

Reply:
As we talked, they can do massive CSS changes and apply after :)